great_expectations/data_context/store/ge_cloud_store_backend.py

```diff
@@ -278,23 +294,33 @@ def _set(
             f"organizations/" f"{organization_id}/" f"{hyphen(resource_name)}",
         )
         try:
-            response = requests.post(url, json=data, headers=self.auth_headers)
-            response_json = response.json()
-
-            object_id = response_json["data"]["id"]
-            object_url = self.get_url_for_key((self.ge_cloud_resource_type, object_id))
-            return GeCloudResourceRef(
-                resource_type=resource_type,
-                ge_cloud_id=object_id,
-                url=object_url,
+            response: requests.Response = requests.post(
+                url, json=data, headers=self.auth_headers
             )
+
+            if isinstance(value, AbstractConfig):
```

Comment:
Remove this check and instead rely on being AbstractConfig
TODO: [ ] - Here we should replace the value from response, call `get` before returning